vite.config.mjs

```diff
@@ -0,0 +1,44 @@
+/// <reference types="vitest" />
```

Comment:
Why did you rename this file?

Reply:
Because of this advice in the terminal 'The CJS build of Vite's Node API is deprecated. See https://vitejs.dev/guide/troubleshooting.html#vite-cjs-node-api-deprecated for more details.'